rundeckapp/grails-spa/packages/ui-trellis/.env.development

```diff
@@ -1,3 +1,3 @@
 VUE_APP_OUTPUT_DIR=../../../grails-app/assets/provided/static
-VUE_APP_DEVTOOL=eval-cheap-module-source-map
+VUE_APP_DEVTOOL=eval-cheap-source-map
```

Comment:
eval-cheap-source-map has better performance than eval-cheap-module-source-map. Open to discussion if team would like to try faster modes:

https://webpack.js.org/configuration/devtool/#devtool